src/context_store/ingestion/deduplicator.py

```
@@ -30,7 +30,7 @@ class DeduplicationAction(str, Enum):
     """重複排除の結果アクション。"""
 
     INSERT = "insert"  # 新規挿入
-    REPLACE = "replace"  # Append-only 置換 (既存を Archive)
+    REPLACE = "replace"  # Append-only 置換（既存を Archive）
```

Ruff (0.15.7)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)